package.json

```diff
@@ -102,7 +102,8 @@
     "test:parity-setup": "mkdir -p test-reports/testcafe && yarn generate-config && yarn workspace v3 dev",
     "test:parity-run": "OKTA_SIW_GEN3=true yarn testcafe -c 4 chrome:headless --reporter spec,xunit:build2/reports/junit/testcafe-results.xml",
     "test:parity-ci": "OKTA_SIW_SKIP_FLAKY=true yarn codegen && run-p -r test:parity-setup test:parity-run",
-    "test:parity-ci-flaky": "OKTA_SIW_GEN3=true OKTA_SIW_ONLY_FLAKY=true yarn test:parity-ci"
+    "test:parity-ci-flaky": "OKTA_SIW_GEN3=true OKTA_SIW_ONLY_FLAKY=true yarn test:parity-ci",
+    "postinstall": "if [[ -z \"$SKIP_POSTINSTALL\" ]]; then patch-package ; fi"
```

Comment:
This command would not work on Windows. 
Tried with GitBash and received error "-z was unexpected at this time."

Maybe we can just strip out `postinstall` command from npm package [here](https://github.com/okta/okta-signin-widget/blob/master/scripts/buildtools/commands/prepack.js#L41) (since we don't export Gen3 anyway in npm package, and I hope this fix is temporary)?

Reply:
Ooh, didn't know about this `prepack` routine. Good idea, will look into it